march_hardware/src/IMotionCube.cpp

```diff
@@ -44,6 +47,7 @@ void IMotionCube::mapMosiPDOs()
   PDOmap pdoMapMOSI = PDOmap();
   pdoMapMOSI.addObject(IMCObjectName::ControlWord);  // Compulsory!
   pdoMapMOSI.addObject(IMCObjectName::TargetPosition);
+  //  pdoMapMOSI.addObject(IMCObjectName::TargetCurrent);
```

Comment:
Commented code

Reply:
How does this even work. TargetCurrent should be compulsory as mosi pdo?!? As miso I don't see the added value.

Reply:
Is no comment anymore